### PR TITLE
Add Configurable support for reusable view extensions

### PR DESCRIPTION
**Pull request by wmcginty**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
 ##### Enhancements
 
+* Add support for `Configurable` types when dequeuing reusable views.
+[Will McGinty](https://github.com/willmcginty)
+[#53](https://github.com/BottleRocketStudios/iOS-UtiliKit/pull/58)
+
 ##### Bug Fixes
 
-* None.
+*
```
Comment (tylermilner) [inline CHANGELOG.md]: Did you intentionally leave this as an empty bullet or should it say "None"?

Comment (wmcginty) [reply CHANGELOG.md]: Was unintentional. Fixed.

**Comment by wmcginty** [comment]
Ahh, Codebeat - 'similar code' operating on two entirely different, unrelated classes.

**Review by tylermilner — APPROVED** [review]

**Comment by wmcginty** [comment]
@tylermilner Any issues with ignoring Codebeat here and merging?

**Pull request merged**